alibuild_helpers/build.py

```diff
@@ -727,7 +727,7 @@ def downloadTask(p):
       logger_handler.setFormatter(
           LogFormatter("%%(asctime)s:%%(levelname)s:%s:%s:%s: %%(message)s" %
                        (mainPackage, p, args.develPrefix if "develPrefix" in args else mainHash[0:8])))
-    if spec["package"] in develPkgs and getattr(syncHelper, "writeStore", None):
+    if spec["package"] in develPkgs and getattr(syncHelper, "writeStore", False):
```

Comment:
Why changing this here? If it is not strictly necessary, can you move it to a separate PR (which can go in immediately)?

Reply:
OK, will do.